Makefile

```diff
@@ -2,6 +2,9 @@
 # Variable definitions
 # ********************
 
+# Environment and python
+VENV_BIN ?= .venv/Scripts/
```

Comment:
The CI fails with the following error:

```
make: .venv/Scripts/pip: Command not found
```

You need to use `VENV_BIN ?= .venv/bin/` here and create a new file, e.g. `windows.mk` with the follwing content:

```
VENV_BIN ?= .venv/Scripts/
include Makefile
```

Then use `make -f windows.mk serve-windev` to start the server.